examples/assets-and-custom-transfer-logic/scripts/0-createAppAsset.js

```diff
@@ -0,0 +1,59 @@
+const {
+  executeTransaction
+} = require('@algorand-builder/algob');
+const { types } = require('@algorand-builder/runtime');
+
+async function run (runtimeEnv, deployer) {
+  const master = deployer.accountsByName.get('master-account');
+  const creator = deployer.accountsByName.get('alice');
+  const bob = deployer.accountsByName.get('bob');
+
+  /** Fund Creator account by master **/
+  const algoTxnParams = {
+    type: types.TransactionType.TransferAlgo,
+    sign: types.SignType.SecretKey,
+    fromAccount: master,
+    toAccountAddr: creator.addr,
+    amountMicroAlgos: 200e6,
+    payFlags: {}
+  };
+  await executeTransaction(deployer, algoTxnParams);
+
+  const asaInfo = await deployer.deployASA('gold', { creator: creator });
+  await deployer.optInToASA('gold', 'bob', {}); // asa optIn for bob
+  console.log(asaInfo);
+
+  /** * Creating Application ***/
+  // initialize app arguments
+  const appArgs = [
+    `int:${asaInfo.assetIndex}`,
+    'int:2' // set min user level(2) for asset transfer ("Accred-level")
+  ];
+
+  const res = await deployer.deploySSC(
```

Comment:
please rename res to `sscInfo` or something else.

Reply:
ok